tests/org.eclipse.passage.loc.features.core.tests/src/org/eclipse/passage/loc/features/core/tests/ConstantsTest.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Nikifor Fedorov <zelenyhleb@gmail.com> - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.loc.features.core.tests;
+
+import static org.junit.Assert.assertEquals;
+
+import org.eclipse.passage.loc.features.core.Features;
+import org.eclipse.passage.loc.features.core.FeaturesCore;
+import org.junit.Test;
+
+public class ConstantsTest {
```

Comment:
There is not much sense in such a test. The idea befind these literals is that a suppier and it's client uses _the same value_, but _the value itself can be changes_ without any disturbance. 

Beseides, all these _public static_ friends are going [to perish](https://www.yegor256.com/2015/07/06/public-static-literals.html). 

But I see the point here is just to initiate the testing bundle. Do not hesitate to create a dummy test like 
```java
@Test 
public void forFurtherDevelopment() {
    assertNotNull("I'm a newborn test bundle, just let me grow.");
}
```